OpenRA.Game/Graphics/Viewport.cs

```diff
@@ -242,6 +242,7 @@ public CPos ViewToWorld(int2 view)
 		{
 			var world = worldRenderer.Viewport.ViewToWorldPx(view);
 			var map = worldRenderer.World.Map;
+			var mapRamp = ((IMapElevation)map).Ramp;
```

Comment:
Likewise here, `Viewport` should be able to handle maps that don't implement `IMapElevation` (this is one of the main usecases in the upstream mods for these interfaces!)